keyboards/projectd/65/projectd_65_ansi/keymaps/via/keymap.c

```diff
@@ -0,0 +1,169 @@
+            }
+            return false; /* Skip all further processing of this key */
+
+        case KC_DND:
+            if (record->event.pressed) {
+                host_system_send(0x9B);
+            } else {
+                host_system_send(0);
+            }
+            return false; /* Skip all further processing of this key */
+
+         case KC_SPOT:
+            if (record->event.pressed) {
+                host_consumer_send(0x221);
+            } else {
+                host_consumer_send(0);
+            }
+            return false; /* Skip all further processing of this key */
+
+          case KC_EJ:
+            if (record->event.pressed) {
+                host_consumer_send(0x0B8);
+            } else {
+                host_consumer_send(0);
+            }
+            return false; /* Skip all further processing of this key */
+
+        default:
+            return true; /* Process all other keycodes normally */
+
+    }
+};
+
+
+enum custom_layers {
+    Layer_0,
+    Layer_1,
+    Layer_2,
+    Layer_3,
+    Layer_4,
+    Layer_5,
+    Layer_6,
+    Layer_7,
+    Layer_8,
+    Layer_9,
+    Layer_10,
+    Layer_11,
+    Layer_12,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* Keymap _BL: Base Layer (Default Layer)
+   */
+[Layer_0] = LAYOUT(
+            QK_GESC,  KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  KC_DEL,
+            KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,  KC_PGUP,
+            KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,            KC_ENT,   KC_PGDN,
+            KC_LSFT,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,            KC_UP,    KC_END,
+            KC_LCTL,  KC_LGUI,  KC_LALT,                      KC_SPC,                                 KC_RALT,  MO(Layer_1),        KC_LEFT,  KC_DOWN,  KC_RGHT),
+[Layer_1] = LAYOUT(
+            KC_GRV,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_DEL,   KC_NO,
+            QK_BOOT,  KC_NO,    KC_NO,    KC_NO,    RGB_TOG,  RGB_MOD,  RGB_RMOD, KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,
+            KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,              KC_NO,    KC_NO,
+            KC_LSFT,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_RSFT,            RGB_VAI,  KC_NO,
+            KC_NO,    KC_NO,    KC_NO,                        KC_NO,                                  KC_NO,    KC_NO,              RGB_SPD,  RGB_VAD,  RGB_SPI),
+[Layer_2] = LAYOUT(
```

Comment:
Especially here.  Any unpopulated layer gets filled automatically with transparencies.  So blank layers are not needed, anymore.

Reply:
we want to use VIA with 13 layers. So, we write like this...

Reply:
I'm in agreement with @drashna, layers without purpose is excessive.

If the intention is for `via` keymap to make available 13 layers in VIA, over the default 4, then please add a `config.h` file inside the `via` directory with the following contents:
```
#pragma once

#define DYNAMIC_KEYMAP_LAYER_COUNT 13
```

Reply:
@dunk2k by doing the process you mentioned is displaying KC_TRNS key in VIA but we need KC_NO key....
Is there any suggested way to do this...thanks

Reply:
@lesshonor We changed this part to KC_TRNS now.